sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationMockTests.cs

```diff
@@ -268,6 +269,26 @@ public async Task ConfiguringTheClient()
             Assert.AreEqual(2, mockTransport.Requests.Count);
         }
 
+        [Test]
+        public async Task AuthorizationHeaderFormat()
+        {
+            var expectedSyntax = "HMAC-SHA256 Credential=(.+)&SignedHeaders=(.+)&Signature=(.+)";
```

Comment:
Do as much of an exact match here as possible, I know the signature is hard because of current date but you can match more than this.